### DISCUSSION TITLE
Is there a way to install additional packages, besides the base LSP

### DISCUSSION THREAD

**Discussion by hahuang65** [mason-org/mason.nvim#785] [Q&A]
For example, for Ruby, `solargraph` can be installed, but it also can be augmented with some gems, like `solargraph-rails`, `rubocop`, `rubocop-rails`, `rubocop-performance`...

Is there anyway to install these without going to `~/.local/share/nvim/mason/packages/` and installing by hand?

**Comment by williamboman** [comment]
Hey! Unfortunately not yet, see #392 for umbrella issue.
